docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+/// tab | leaf3
+
+```{.srl .code-scroll-lg}
+--{ + running }--[  ]--
+A:leaf2# info interface irb0
+    interface irb0 {
+        admin-state enable
+        subinterface 10 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 172.16.10.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    proxy-arp true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+/// tab | leaf4
+
+```{.srl .code-scroll-lg}
+--{ + running }--[  ]--
+A:leaf2# info interface irb0
+    interface irb0 {
+        admin-state enable
+        subinterface 10 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 172.16.10.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    proxy-arp true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+        subinterface 20 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 172.16.20.254/24 {
+                    anycast-gw true
+                }
+                arp {
+                    learn-unsolicited true
+                    host-route {
+                        populate dynamic {
+                        }
+                        populate evpn {
+                        }
+                    }
+                    evpn {
+                        advertise dynamic {
+                        }
+                    }
+                }
+            }
+            anycast-gw {
+                anycast-gw-mac 00:00:5E:00:53:00
+            }
+        }
+    }
+```
+
+///
+
+There is a lot going on here, so let's breakdown some of the configuration options:
+
+`anycast-gw [true|false]`
```

Comment:
from Jorge

..derived from the VRRP MAC address group range, as specified by RFC9135.